packages/@react-spectrum/table/docs/TableView.mdx

```diff
@@ -511,7 +514,7 @@ function AsyncSortTable() {
         items: items.sort((a, b) => {
           let first = a[sortDescriptor.column];
           let second = b[sortDescriptor.column];
-          let cmp = (parseInt(first) || first) < (parseInt(second) || second) ? -1 : 1;
+          let cmp = collator.compare((parseInt(first) || first), (parseInt(second) || second));
```

Comment:
This looks like it has problems when sorting `height`/`mass`

Reply:
Ah, this is probably since the collator by default doesn't include the `{numeric: true}` option. You can trying passing that to the `useCollator` call above. Additionally, you can probably remove the `parseInt` since the collator expects strings

Reply:
Missing option has been added.